test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -862,6 +863,171 @@ public void testNestedSecondaryIndexesRepeatedFieldFollowedByNonPrimitive() thro
         }
     }
 
+    /**
+     * Example to see how nested secondary indexes work when a secondary index is set on a 'oneOf' field.
+     * This test combines all possibilities of secondary indexes on 'oneOf' properties:
+     * (1) 'oneOf' index nested in a repeated field
+     *      (a) Index primitive type
+     *      (b) Index non-Primitive type
+     *      (c) Index repeated Primitive
```

Comment:
we may want to reword it to something like 'index repeated primitive inside a oneOf' to make it clearer?

Reply:
that's what I tried to say on L.869... all these are within a oneOf, and then internally the index can be a primitive, non-primitive, or a repeated primitive.. let me reword to make it clear..

Reply:
I changed it a bit to avoid confusion.